### PR TITLE
Implement the new Revamp designs for the add wallet page

### PR DESCRIPTION
**Pull request by neuodev**
Screen Shot 2023-02-20 at 2 59 08 PM

### CODE REVIEW

File: packages/yoroi-extension/app/components/wallet/add-wallet-revamp/AddWalletCard.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import type { Node } from 'react';
+import { Box, Typography } from '@mui/material';
+import styles from './AddWalletCard.scss';
+
+type Props = {|
+    label: string,
+    imageSrc: string,
+|};
+
+export default class AddWalletCard extends Component<Props> {
```
Comment (nistadev) [inline packages/yoroi-extension/app/components/wallet/add-wallet-revamp/AddWalletCard.js]: I think we better use a functional component here

File: packages/yoroi-extension/app/components/wallet/add-wallet-revamp/AddWalletPageContent.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        defaultMessage: 'Restore existing wallet',
+    },
+    connectHardwareWallet: {
+        id: 'wallet.add.page.revamp.connectHardwareWallet',
+        defaultMessage: 'Connect hardware wallet',
+    },
+});
+
+@observer
+export default class AddWalletPageContent extends Component<{||}> {
```
Comment (nistadev) [inline packages/yoroi-extension/app/components/wallet/add-wallet-revamp/AddWalletPageContent.js]: Use a functional component and add `injectIntl`

File: packages/yoroi-extension/app/containers/wallet/AddWalletPageRevamp.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+// @flow
+import { Component } from 'react';
+import type { Node } from 'react';
+import AddWalletPageHeader from '../../components/wallet/add-wallet-revamp/AddWalletPageHeader';
+import { Box } from '@mui/material';
+import AddWalletPageContent from '../../components/wallet/add-wallet-revamp/AddWalletPageContent';
+
+export default class AddWalletPageRevamp extends Component {
```
Comment (nistadev) [inline packages/yoroi-extension/app/containers/wallet/AddWalletPageRevamp.js]: Use functional component

File: packages/yoroi-extension/app/components/wallet/add-wallet-revamp/AddWalletPageHeader.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+const messages: * = defineMessages({
+    subtitle: {
+        id: 'wallet.add.page.revamp.subtitle',
+        defaultMessage: 'Light wallet for Cardano assets',
+    }
+});
+
+@observer
+export default class AddWalletPageHeader extends Component<{||}> {
```
Comment (nistadev) [inline packages/yoroi-extension/app/components/wallet/add-wallet-revamp/AddWalletPageHeader.js:21]: Use functional component

**Review by nistadev — CHANGES_REQUESTED** [review]

**Review by a-hosani — APPROVED** [review]

**Comment by neuodev** [comment]
[autoscirpt] Jira Ticket: **[implement the new Revamp designs for the add wallet page](https://emurgo.atlassian.net/browse/YOEXT-524)**

[](__JIRA_COMMENT_FLAG)

**Review by a-hosani — APPROVED** [review]

**Review by Nebyt — APPROVED** [review]
Checked. :+1:

File: packages/yoroi-extension/app/components/wallet/add-wallet-revamp/AddWalletCard.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+import { Box, Typography } from '@mui/material';
+import styles from './AddWalletCard.scss';
+
+type Props = {|
+  label: string,
+  imageSrc: string,
+  onClick(): void,
+|};
+
+export default function AddWalletCard(props: Props): Node {
```
Comment (nistadev) [inline packages/yoroi-extension/app/components/wallet/add-wallet-revamp/AddWalletCard.js:12]: Add wallet card seems not to fit into a generic component. We use this type of card on other funnels, eg. Restore. I think we should change the name and structure to fit this more generic purpose.

**Review by nistadev — CHANGES_REQUESTED** [review]

**Review by zuzunker — APPROVED** [review]

**Pull request merged**